_posts/2015-10-19-introduction_to_clojurescript.md

```diff
@@ -0,0 +1,269 @@
+---
+layout: post
+title: Introduction to ClojureScript
+category: Programming
+tags:
+  - JavaScript
+  - Node.js
+  - ClojureScript
+---
+
+# Introduction to ClojureScript
+In this article, we will be discussing ClojureScript a compile-to-javascript language. We'll be looking at the benefits of working with ClojureScript and how you can quickly get set up to use NPM and your favorite Node.js libraries.
```

Comment:
"we will be discussing" and "We'll be looking" - pick contractions or expanded and use consistently. I suggest using contractions for blog posts for better readability